lib/components/molecules/AccordionBasic/AccordionBasic.story.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  accordionType1,
+  accordionType2,
+  accordionType3,
+} from './AccordionBasic.mock';
+import { unorderedList } from '../List/List.mock';
+import List from '../List';
+
+// Import Styled Component to showcase variations
+import AccordionBasic from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
```

Comment:
Remove this // $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code